CharacterCreation/01_Races.txt

```diff
@@ -67,41 +67,42 @@ Stat Modifiers: Flexible - +1 DEX, +1 to PER
 Language: Racial + ECSC Standard
 
 Racial Traits:
-Camouflage: Choose a natural terrain type (Jungle, mountain, plaiins, desert, 
+Camouflage: Choose a natural terrain type (Jungle, mountain, plains, desert, 
 arctic, urban, etc.). While in this terrain you have a +20% buff to 
 stealth.
 Stalker: Gain Stealth and Awareness skills at 15 if you take them as level 1 
 skills, instead of the normal 10.
 Low-Light Vision: can see in moonlight as well as if it were day-light. only 
 requires dim light to see well.
 Sprinter - +1m movement speed when charging, retreating,  or otherwise moving 
-under preasure.
+under pressure.
 
 //============================================================
 Detachment A Security Drone
 Society: Slave soldiers. Squad families.
 Security Drones are specialized drones that have been granted a greater AI 
-capacity to learn and think independantly. They are native to ECSC Compound X 
+capacity to learn and think independently. They are native to ECSC Compound X 
 experimentation laboratory.
 They have a steel reinforced frame, and are usually armor plated. The 
 coloration of plating can range anywhere in the natural colors and camouflages,
-or Flat Grey, or Flat Black. They have a single occular sensor situated on a 
-smallish sqaure head. They generally stand somewhere in the 6 foot to 7 foot 
+or Flat Grey, or Flat Black. They have a single ocular sensor situated on a 
+smallish square head. They generally stand somewhere in the 6 foot to 7 foot 
 range, and weigh between 400 and 700 pounds.
 
 Favored Class: Marksman, Soldier, Adept, Priest, Mechanic.
 
 Qualities: 10RP
-Type: Robotic. Does not need to eat, breath, or sleep. Must spend 6 hours to 
-recharge after a 24 hour period. this can be done via a self-contained 
-reactor. Not affected by mind-influence, succeptable to hack attempts.
+Type: Cybernetic
 Size: Medium+
 Speed: Normal
 Starting Stat Points: 40
 Stat Modifiers: INT + 1, FORT + 2. LUCK -3.
 Language: Basic
 
 Racial Traits:
+Robotic. Does not need to eat, breath, or sleep. Must spend 6 hours to 

```

Comment:
I know it's a small thing.. Can you change "Robotic." to "Robotic:" ?

Reply:
Can do, will do :)